vehicle/psa/identity.go

```diff
@@ -1,24 +1,46 @@
 package psa
 
 import (
-	"context"
+	"encoding/base64"
+	"errors"
 	"fmt"
+	"net/http"
+	"net/url"
+	"strings"
+	"sync"
+	"time"
 
+	"github.com/evcc-io/evcc/server/db/settings"
 	"github.com/evcc-io/evcc/util"
+	"github.com/evcc-io/evcc/util/oauth"
 	"github.com/evcc-io/evcc/util/request"
 	"golang.org/x/oauth2"
 )
 
+var (
+	mu sync.Mutex
+)
+
 type Identity struct {
 	*request.Helper
 	oc *oauth2.Config
 	oauth2.TokenSource
+	mu    sync.Mutex
+	log   *util.Logger
+	brand string
+	realm string
```

Comment:
Wofür brauchts das noch?

Reply:
mu/log hab ich mehr oder weniger vom Mercedes Code übernommen. Könnte man natürlich aber in Frage stellen ob es das braucht...
Ersteres wird aktuell auch in der RefreshToken Funktion genutzt. Ob man das dafür mitführen muss, wäre natürlich die Frage...

Reply:
Frage bezog sich nur auf Realm.